platform/default/mbgl/storage/offline_database.cpp

```diff
@@ -49,7 +49,8 @@ void OfflineDatabase::ensureSchema() {
             case 1: break; // cache-only database; ok to delete
             case 2: migrateToVersion3(); // fall through
             case 3: migrateToVersion4(); // fall through
```

Comment:
Yes. In fact, we should require that every new migration be accompanied by the addition of a test case that tests migrating from the prior version. In local development, migrations are a rarely-exercised code path.